test/e2e/framework/service/jig.go

```diff
@@ -901,10 +901,17 @@ func (j *TestJig) checkNodePortServiceReachability(svc *v1.Service, pod *v1.Pod)
 // checkExternalServiceReachability ensures service of type externalName resolves to IP address and no fake externalName is set
 // FQDN of kubernetes is used as externalName(for air tight platforms).
 func (j *TestJig) checkExternalServiceReachability(svc *v1.Service, pod *v1.Pod) error {
+	// NOTE(claudiub): Windows does not support PQDN.
+	svcName := svc.Name
+	if framework.NodeOSDistroIs("windows") {
+		svcName = fmt.Sprintf("%s.%s.svc.%s", svcName, svc.Namespace, framework.TestContext.ClusterDNSDomain)
```

Comment:
Just a question.
Can we just change here to
```
svcName = fmt.Sprintf("%s.%s.svc.%s", svcName, svc.Namespace, framework.TestContext.ClusterDNSDomain)
```
without considering Linux or Windows?

This function is used in conformance tests as the corresponding issue says, and the conformance test requirements[1] contains `where possible, it does not depend on outputs that change based on OS (nslookup, ping, chmod, ls)`
That doesn't seems mandatory but it seems better to use the same testing way as possible.

[1]: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-test-requirements

Reply:
Done